docs/APIRef.DetoxCLI.md

```diff
@@ -102,6 +102,7 @@ Initiating your test suite[^1].
 | --use-custom-logger                           | Use Detox' custom console-logging implementation, for logging Detox (non-device) logs. Disabling will fallback to node.js / test-runner’s implementation (e.g. Jest / Mocha).<br />_Default: true_ |
 | --force-adb-install                           | Due to problems with the `adb install` command on Android, Detox resorts to a different scheme for installing APKs. Setting true will disable that and force usage of `adb install`, instead.<br/>This flag is temporary until the Detox way proves stable.<br/>_Default: false_ |
 | --inspect-brk                                 | Uses [node’s --inspect-brk](https://nodejs.org/en/docs/guides/debugging-getting-started/#enable-inspector) flag to let users debug the jest/mocha test runner <br />_Default: false_ |
+| --optimize-app-install                        | Uses an optimized method for installation where a hash is stored on the device allowing future app installation to be skipped when possible.<br />_Default: false_                                                                                                                                        |
```

Comment:
Only Android

Reply:
Also I wouldn't elaborate on implementation (stored hash etc..), only saying that it depends whether the APK has changed.